BasicBot.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+import discord
+from ButtonViews import ButtonView
```

Comment:
Modules should always be snake_case. Please change it to button_views (button_views.py). The class name is perfect as is though!